packages/devtools_app/lib/src/shared/memory/adapted_heap_data.dart

```diff
@@ -117,6 +117,8 @@ class AdaptedHeapData {
 
   late DateTime created;
 
+  String snapshotName = '';
```

Comment:
should this be late final?

Reply:
It is initialized after creation and i do not want anything to fail if it is accessed before initialization.